tools_iuc.yaml

```diff
@@ -4,6 +4,10 @@ install_resolver_dependencies: true
 install_tool_dependencies: false
 
 tools:
+  - name: maaslin2
+    owner: iuc
+    tool_panel_section_label: Microbiom R Analysis
```

Comment:
you are not free to choose a panel section ... otherwise we would have 1000 of categories